crates/cli/tests/reference/anyref-import-catch.bg.js

```diff
@@ -1,8 +1,32 @@
-let wasm;
-export function __wbg_set_wasm(val) {
-    wasm = val;
+//#region exports
+
+export function exported() {
+    const ret = wasm.exported();
+    if (ret[1]) {
+        throw takeFromExternrefTable0(ret[0]);
+    }
+}
+//#endregion
+
+//#region wasm imports
+export function __wbg___wbindgen_throw_dd24417ed36fc46e(arg0, arg1) {
+    throw new Error(getStringFromWasm0(arg0, arg1));
+}
+export function __wbg_foo_9ceaa09e9e1e3047() { return handleError(function () {
+    foo();
+}, arguments) }
```

Comment:
Removing the trailing semicolon here is correct, but formally speaking we should have always had a semicolon after the return statement.

Reply:
Done.